internal/pkg/service/buffer/api/service/service.go

```diff
@@ -21,7 +21,7 @@ import (
 	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/api/gen/buffer"
 	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/api/service/mapper"
 	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/file"
-	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/stats"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/statistics"
```

Comment:
`stats` je bezny nazov premennej a koliduje to na viacerych miestach s nazvom tohto pkg, tak to navrhujem zmenit.

https://go.dev/blog/package-names
```
Don’t steal good names from the user. 
Avoid giving a package a name that is commonly used in client code. 
For example, the buffered I/O package is called bufio, 
not buf, since buf is a good variable name for a buffer.
```